src/shared/actions/members.js

```diff
@@ -94,6 +95,113 @@ async function getStatsDone(handle, uuid, tokenV3) {
   return { data, handle, uuid };
 }
 
+/**
+ * The maximum number of challenges to fetch in a single API call. Currently,
+ * the backend never returns more than 50 challenges, even when a higher limit
+ * was specified in the request. Thus, this constant should not be larger than
+ * 50 (otherwise the frontend code will miss to load some challenges).
+ */
+const PAGE_SIZE = 50;
+
+/**
+ * Private. Loads from the backend all challenges matching some conditions.
+ * @param {Function} getter Given params object of shape { limit, offset }
+ *  loads from the backend at most "limit" challenges, skipping the first
+ *  "offset" ones. Returns loaded challenges as an array.
+ * @param {Number} page Optional. Next page of challenges to load.
+ * @param {Array} prev Optional. Challenges loaded so far.
+ */
+function getAll(getter, page = 0, prev = null) {
+  /* Amount of challenges to fetch in one API call. 50 is the current maximum
+   * amount of challenges the backend returns, event when the larger limit is
+   * explicitely required. */
```

Comment:
why these are here (this, and the following few functions related to loading of challenges)? it looks to me as copy/paste of code from `challenges` actions / reducers / service, we should just rely on that, and not duplicate it here.